ogcapi-features/README.md

```diff
@@ -106,37 +106,32 @@ If the target of a Link's `type` is unknown, `type` SHOULD be set to `applicatio
-| ----------- | -------- | -------------- | --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `canonical` | various  | various        | STAC API - Core | Provides the preferred paths to get to STAC Collection and Item objects, if they differ from the URL that was used to retrieve the STAC object and thus duplicate other content. This can be useful in federated catalogs that present metadata that has a different location than the source metadata. |
+| **rel**     | **href**                      | **Media Type**   | **From**                                    | **Description**                                                                                                                                                                                                                                                                                         |
+| ----------- | ----------------------------- | ---------------- | ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `root`      | `/`                           | application/json | STAC API - Features, STAC API - Collections | **REQUIRED** The root URI                                                                                                                                                                                                                                                                               |
+| `self`      | `/collections/{collectionId}` | application/json | OAFeat                                      | **REQUIRED** Self reference                                                                                                                                                                                                                                                                             |
+| `parent`    | `/`                           | application/json | OAFeat                                      | Parent reference, usually the root Catalog                                                                                                                                                                                                                                                              |
```

Comment:
```suggestion
| `parent`    | `/`                           | application/json | STAC Item                                   | Parent reference, usually the root Catalog                                                                                                                                                                                                                                                              |
```

OGC API specs don't mention `parent` as far as I can tell (http://docs.opengeospatial.org/DRAFTS/17-069r5.html#_link_relations).

Reply:
correct, that's a pre-existing error -- glad we can fix it now.